include/constants/battle_script_commands.h

```diff
@@ -173,6 +173,8 @@
 #define VARIOUS_DESTROY_ABILITY_POPUP           102
 #define VARIOUS_TOTEM_BOOST                     103
 #define VARIOUS_TRY_ACTIVATE_GRIM_NEIGH         104
+#define VARIOUS_PHOTON_GEYSER_CHECK          105
+#define VARIOUS_SHELL_SIDE_ARM_CHECK          106
```

Comment:
Throw in a few empty spaces in these entries to align the list of number IDs properly.